### PR TITLE
webmockr test

### PR DESCRIPTION
**Pull request by maelle**
cf https://github.com/ropensci/webmockr/issues/94

### CODE REVIEW

**Comment by maelle** [comment]
The failures on Ubuntu are due to failures in installing dependencies.

File: R/utils.R [hunk 1]
```diff
@@ -203,12 +203,23 @@ replace_plus <- function(x){
...
+}
+
 get_status <- function(){
-  client <- crul::HttpClient$new(url = "https://api.openaq.org/status")
-  status <- client$get()
-  status <- suppressMessages(status$parse())
-  status <- jsonlite::fromJSON(status)
-  return(status$results$healthStatus)
+  client <- crul::HttpClient$new(url = status_url())
+  status <- try(client$retry("get"), silent = TRUE)
```
Comment (sckott) [inline R/utils.R]: this looks good to me, except that this doesn't discern between crul failing for the reason you are expecting, for the webmockr stub, vs. another reason.

Comment (maelle) [reply R/utils.R]: Right, it'd be clever to write if status != 200 or so I suppose.

Comment (maelle) [reply R/utils.R]: @sckott can you please have a look at the new diff?

Comment (sckott) [reply R/utils.R]: yes

File: R/utils.R [hunk 2]
```diff
@@ -203,12 +203,23 @@ replace_plus <- function(x){
...
-  status <- suppressMessages(status$parse())
-  status <- jsonlite::fromJSON(status)
-  return(status$results$healthStatus)
+  client <- crul::HttpClient$new(url = status_url())
+  status <- client$retry("get")
+
+  if (status$status_code >= 400) {
+    return("red")
+  } else {
+    status <- suppressMessages(status$parse())
```
Comment (sckott) [inline R/utils.R:218]: you probably want to set encodign for `$parse()` - `UTF-8` i'd assume?

File: R/utils.R (hunk 1 above)
Comment (sckott) [reply R/utils.R]: looks good

**Comment by maelle** [comment]
thanks @sckott

**Pull request merged**